build/rubygem-simp-cli.spec

```diff
@@ -129,6 +129,14 @@ EOM
 %doc %{gemdir}/doc
 
 %changelog
+* Tue Jun 11 2019 Chris Tessmer <chris.tessmer@onyxpoint.com> - 5.0.0
```

Comment:
We haven't released a version with ``simp environment`` yet, so you don't have to note the bug fixes here.